### ISSUE TITLE
Sync manga follow

### ISSUE THREAD

**Issue by kukaburo** [CarlosEsco/Neko#1371]
### Steps to reproduce

1. Mangadex login
2. Sync mangadex follow

### Expected behavior

1605 manga imported

### Actual behavior

1179 manga imported

### Neko version

2.12.9.1

### Android version

Android 12

### Device

Xiaomi Redmi Note 9 Pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
My best guess is some of them aren't "followed" meaning they don't have the bell

**Comment by kukaburo** [comment]
You're right. Is there any solution to make all of them followed or something that can export all of the library to neko?

**Comment by CarlosEsco** [comment]
Not currently. I am waiting for the MD list redesign

**Comment by kukaburo** [comment]
Okay. Thank you...
